docker/1.0-1/final/Dockerfile_arm.cpu

```diff
@@ -1,4 +1,4 @@
-FROM sklearn-base:1.0-1-cpu-py3-arm64
+FROM sklearn-base:1.0-1
```

Comment:
Based on the buildspec it looks like the base image will have `-arm64` suffix? https://github.com/aws/sagemaker-scikit-learn-container/blob/master/ci/buildspec_arm.yml#L29

Reply:
Yes you're right, good catch